rules/chrome.json

```
@@ -5,9 +5,9 @@
   "dependencies": [
     {
       "pre_install": [
-        { "command": "apt-get install -y gnupg wget" },
-        { "command": "if [ -z $(which google-chrome) ]; then wget -O /tmp/google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb; fi" },
-        { "command": "if [ -z $(which google-chrome) ]; then DEBIAN_FRONTEND='noninteractive' apt-get install -y /tmp/google-chrome.deb; fi" }
+        { "command": "[ $(which google-chrome) ] || apt-get install -y gnupg wget" },
+        { "command": "[ $(which google-chrome) ] || wget -O /tmp/google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb" },
+        { "command": "DEBIAN_FRONTEND='noninteractive'; [ $(which google-chrome) ] || apt-get install -y /tmp/google-chrome.deb" }
```

I think `DEBIAN_FRONTEND` isn't being set for the install command, which is causing one of the CI tests to hang at an interactive prompt. How about..
```suggestion
        { "command": "[ $(which google-chrome) ] || DEBIAN_FRONTEND='noninteractive' apt-get install -y /tmp/google-chrome.deb" }
```

Alternatively, an `export DEBIAN_FRONTEND='noninteractive'` should fix it, but that'll be trickier to clean up after.